### PR TITLE
Fix build chain

### PR DESCRIPTION
**Pull request by llambeau**
The folllowing PR:

- Makes sure the npm package is build before being published
- Ignores the src/ folder for the NPM package
- Refreshes outdated dependencies
- Disables saucelabs integration as it is highly unreliable

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "finitio",
-  "version": "0.1.0",
+  "version": "0.2.1",
```
Comment (blambeau) [inline bower.json]: Why not 0.2.0 exactly?

Comment (llambeau) [inline bower.json]: Because we have already 0.2.0 apparently: https://www.npmjs.com/package/finitio

Comment (blambeau) [inline bower.json]: Btw, I would actually go for 1.0.0. Finitio.js is used in production for a while and changing all dependencies at once might potentially hurt some projects. Let announce it clearly with a major version bump.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "finitio",
-  "version": "0.1.0",
+  "version": "0.2.1",
```
Comment (blambeau) [inline package.json]: Same remark

File: bower.json (hunk 1 above)
Comment (llambeau) [inline bower.json]: Ok for 1.0.0 then

File: src/finitio/support/heading.coffee [hunk 3]
```diff
@@ -34,7 +34,7 @@ class Heading
     @options ?= {}
     unless $u.isObject(@options)
       $u.argumentError("Hash of options expected")
-    @options = $u.extend({}, DEFAULT_OPTIONS, options)
+    @options = $u.extend({}, DEFAULT_OPTIONS, @options)
```
Comment (blambeau) [inline src/finitio/support/heading.coffee]: Oh oh, and the tests did not fail before? Looks like a test is missing here.

**Pull request merged**